docs/html/development/contributing.rst

```diff
@@ -41,7 +42,7 @@ Automated Testing
 All pull requests and merges to 'master' branch are tested using `Travis CI`_
 and `Appveyor CI`_ based on our `.travis.yml`_ and `.appveyor.yml`_ files.
```

Comment:
I think we can remove the reference of Appveyor from the docs, since it was removed in https://github.com/pypa/pip/pull/7564

Reply:
> I think we can remove the reference of Appveyor from the docs, since it was removed in #7564

A follow up PR updating this sentence would be great.

Reply:
Posted https://github.com/pypa/pip/pull/8112 for this